grails-app/domain/com/unifina/domain/security/SecUser.groovy

```diff
@@ -20,17 +20,24 @@ class SecUser implements Userish {
 	Set<Key> keys
 	Set<Permission> permissions
 
+	// dateCreated is the date when account is created.
+	Date dateCreated
+	// lastLogin is the date when last successful login was made.
+	Date lastLogin
+
 	static hasMany = [permissions: Permission, keys: Key]
 
 	static constraints = {
 		username blank: false, unique: true, validator: UsernameValidator.validate
 		password blank: false
 		name blank: false
+		lastLogin nullable: false
```

Comment:
I know I said otherwise on Slack, but on second thought ... why not just leave `lastLogin` as nullable? Having a `null` value there maybe makes more sense after all than `1970-01-01 00:00:00`. What do you think?

If nullable then this row can be removed.